array.c

```diff
@@ -4631,6 +4631,56 @@ rb_ary_and(VALUE ary1, VALUE ary2)
     return ary3;
 }
 
+/*
+ *  call-seq:
+ *     ary.overlaps?(ary2)   -> true or false
+ *
+ *  Returns +true+ if the intersection of +ary+ and +ary2+ is non empty, else returns false. 
+ *
+ *  This is equivalent to (ary & ary2).empty?
```

Comment:
Isn't it `!(ary & ary2).empty?` ?

Reply:
Of course - I meant to type `.any?`, as in the description here.